#Protocol

##1.	PCR of vector and target gene

* Amplify the vector, increasing its amount to make it detectable and analyzable.
<center>Table 1 Reaction system</center>

|Component|Volume|
|:--|:--|
|Phanta SE Super-Fidelity DNA Polymerase|1µL | 
|Phanta SE Buffer|25ul|
|Template of vector|XµL|
|Forward primer for vector|2µL|
|Reverse primer for vector|2µL|
|Sterilized water|Up to 50µL|

<center>Table 2 Reaction procedure</center>

|Cycle steps|Temperature|Time|Cycles|
|:--|:--|:--|:--|
|Predegeneration|98℃|30 sec|  |
|Transgender|98℃|10 sec|  |
|Annealing|Tm|5 sec|25-35|
|Extension|72℃|5 - 10 sec/kb| |
|Completely stretch|72℃|1 min| |

* Steps:
1.	Thaw the components on ice and add each group as needed to dispense the PCR reaction system.
2.	Gently centrifuge and mix well. 
3.	Transfer the PCR tubes to the PCR instrument, set the parameters, and start the polymerase chain reaction.
4.	Return the reagents to 4℃ or 12℃ for storage immediately after us.

##2.	Error-prone PCR
* Amplify the vector, increasing its amount to make it detectable and analyzable.

|Component|Volume|
|:--|:--|
|Taq enzyme|25μl| 
|Template|1μl| 
|Primer|Forward 2μl Reverse 2μl| 
|Sterile water|Refill to 50μl| 

* Refer to the table below to set up a random mutation PCR reaction system

|Reagents|Final concentration|Volume|Volume|
|:--|:--|:--|:--|
|Sterile water| |(13.2-x)μl|(33-y)μl|
|RandomMut buffer (10X)|1X|2μl|5μl |
|Mutation enhancer (10X)|1X|2ul|5ul|
|dNTP (2.5mM each)|0.25mM|2μl|5μl |
|template DNA|0.2pg/μl-20ng/μl|xμl|yu |
|Primer mixture (10μM each)|0.2μM each|0.4μl|1μl|
|RandomMut DNA polymerase|-|0.4μl|1μl|
|Total Volume|-|20μl|50μl|

|Mutation rate>|Mutations/kb|Target amount/20ul|Target concentration|
|:--|:--|:--|:--|
|Low|0-4.5 |20-400ng|1-20ng/μl|
|Medium|4.5-9|2-20ng|0.1-1ng/μl|
|High|9-16|40pg-2ng|2-100pg/μl|

|Cycle steps|Temperature|Time|Cycles|
|:--|:--|:--|:--|
|Predegeneration|94℃ |3 min|  |
|Transgender|94℃|30 sec|  |
|Annealing|Tm|30 sec|25-30|
|Extension|72℃|1min/kb| |
|Completely stretch|72℃|10 min|  |

*	Steps:
1.	Ensure the accurate addition of PCR reaction system and reagents, including template DNA, primers, dNTP, etc. The concentration and proportion of these components should be precise, without missing or adding them incorrectly
2.	During PCR amplification, setting the annealing temperature is one of the most important steps in the PCR cycle.
3.	The cycling process should ensure that the PCR instrument is sufficiently cooled and operates normally to avoid non-specific products.

##3.	Agarose Gel Electrophoresis
*	Separate DNA fragments of different sizes and compare their positions with the marker to check whether the products are the desired ones.
*	Steps:
1.	Place the gel tray in the proper position in the gel box, and place the comb in the correct position.
2.	Mix agarose and 1 × TAE buffer in a ratio of 0.8g to 100mL as needed, pour into a 250mL conical flask, heat and boil the flask in a microwave until the agarose is completely dissolved.
3.	Add staining solution to the solution in a ratio of 2 µ LGel RED: 100mL 1 × TAE.
4.	Pour the solution into the tray.
5.	Allow the gel cool and solidify, then pull out the comb.
6.	Place the gel into the electrophoresis apparatus, and enough 1 × TAE buffer solution is required.
7.	Add 10 x DNA loading buffer to the sample and mix, then transfer the mixture into the gel hole with a pipette.
8.	Turn on the power and run at 120V for 20 minutes.

##4.	DNA Fragment Gel Recovery

After nucleic acid gel electrophoresis was verified, the bands were observed under UV light to see if they were the same size as the target fragment. The target fragments were recovered according to the operation of a DNA purification kit (FastPure○R Gel DNA Extraction Mini Kit, Nanjing Novozymes Bioscience and Technology Co., Ltd.), and stored at -20℃ in a low-temperature refrigerator for spare use.

##5.	Ligation of vector and target gene
*	Construct plasmid using the target gene and the vector using Gibson assembly.
<center>Table 3 Plasmid ligation system</center>
|Component|Volume|
|:--|:--|
|Sterilized water|Up to 10µL| 
|2x CE Mix|5µL| 
|Vector sample|Calculate according to instructions| 
|Target gene sample|Calculate according to instructions| 

*	Steps:
1.	Calculate the sample volumes of the target gene and the vector according to the formulas: Target gene volume = bp x 0.04 / concentration, Vector volume = bp x 0.02 / concentration.
2.	Calculate the volume of sterilized water required to make up to 10 μl.
3.	Add the sterilized water, 2x CE Mix, vector sample and Target gene sample into the EP tube.
4.	Put the EP tube in a centrifuge for centrifugation.
5.	Put the EP tube in the PCR instrument, set it to 50 ˚C for 15 or 30 minutes.

##6.	Chemical Transformation of E. coli with Recombinant Plasmid
*	Receptive E. coli cells are transformed with a recombinant plasmid containing target gene.
<center>Table 4 Chemical conversion system</center>
|Component|Volume|
|:--|:--|
|Receptive E. coli cell|100µL| 
|Recombinant plasmid|10µL| 
|LB liquid medium|800µL| 

*	Steps:
  (The following procedures involving bacteria should be conducted in a biosafety cabinet)
 	1.	Add10μl of the recombinant plasmid DNA to100 μl of competent E. coli cells, and place it on ice for 30 minutes.
2.	After 30 minutes on ice, subject the cells to a heat shock at 42°C for 90 seconds.
Then immediately return the cells to ice for 2-3 minutes.
3.	Add 800 µL of antibiotic-free LB liquid medium to the plasmid-added sensory state and incubate for 1 h at 37°C in a shaker
4.	Centrifuge the cells at a low speed to pellet the cells.
5.	Discard 800 μl of  the supernatant
6.	6.	Resuspend the cell pellet in the remaining 100 μl of medium by pipetting up and down.
7.	Spread the mixture evenly onto LB agar plates containing antibiotics using a spreader.
8.	Incubate the plates overnight at 37°C.

##7.	Colony PCR and Verification of E. coli Transformation
*	Verifying the presence of the recombinant plasmid in transformed E. coli colonies using colony PCR and subsequent electrophoresis.

<center>Table 5 Reaction system</center>
|Component|Volume|
|:--|:--|
|Upstream primer|0.2µL| 
|Downstream primer|0.2µL| 
|Sterilized water|Up to 5µL| 
|2×Rapid Taq  Mix|5µL| 
|Mud or liquid of bacteria|Xul| 

*	Steps:
(The following procedures involving bacteria should be conducted in a biosafety cabinet)
1.	Using a sterile toothpick or pipette tip, pick a small amount of bacterial colony and mix it with the PCR reaction mixture.
2.	Place the PCR tubes in a thermal cycler and run the program:

<center> Table 6 Reaction procedure</center>
|Cycle steps|Temperature|Time|Cycles|
|:--|:--|:--|:--|
|Predegeneration|95℃| 10 min| |
|Transgender|95℃|15 sec| |
|Annealing|Tm|15 sec|25-35 |
|Extension|72℃|15 - 30 sec/kb| |
|Completely stretch|72℃|5 min| |

3.	Conduct agarose gel electrophoresis on the PCR products, and check if the result is as expected 

##8.	Plasmid extraction
*FastPure® Plasmid Mini Kit (Vazyme) was used for rapid purification and extraction of plasmid DNA.
*Steps: 
1. Label and transfer 2-4 mL of bacterial culture to a centrifuge tube under aseptic conditions. Balance the tube for mass distribution, place it symmetrically in the centrifuge, check for proper sealing, set the speed to 12000 rpm for 1 min,discard the culture medium, invert the tube on absorbent paper to remove residual liquid, and retain the pellet. 
2. Add 250 μL of Buffer P1 (Buffer P1 with RNase A added) to the centrifuge tube containing the bacterial pellet. Mix thoroughly using a pipette or vortex oscillation. 
3. Add 250 μL of Buffer P2 to the mixture from step 2. Gently invert the tube 8-10 times to ensure thorough lysis of the bacterial cells until a clear solution is formed. 
4. Add 350 μL of Buffer P3 to the mixture from step 3. Immediately invert the tube gently 8-10 times to thoroughly neutralize the solution with Buffer P2. A white flocculent precipitate should appear. Centrifuge at 12000 rpm (13,400 × g) for 10 min.
 5. Place the FastPure DNA Mini Columns in a 2 mL Collection Tube. Carefully transfer the supernatant from step 4 to the column using a pipette, being careful not to aspirate the precipitate. Centrifuge at 12000 rpm (13,400 × g) for 30-60 sec. Discard the waste liquid from the collection tube and return the column to the collection tube.
 6. Add 500 μL of Buffer PW1 to the column. Centrifuge at 12000 rpm (13,400 × g) for 30-60 sec. Discard the waste liquid and return the column to the collection tube. 
7. Add 600 μL of Buffer PW2 (check if it has been diluted with anhydrous ethanol) to the column. Centrifuge at 12000 rpm (13,400 × g) for 30-60 sec. Discard the waste liquid and return the column to the collection tube. Repeat this step once. 
8. Place the column back into the collection tube. Centrifuge at 12000 rpm (13,400 × g) for 1 min to dry the column and remove any residual wash liquid.
 9. Place the column in a new sterile 1.5 mL centrifuge tube. Add 30-100 μL of sterile ddH2O to the center of the column membrane. Incubate at room temperature for 2 min, then centrifuge at 12000 rpm (13,400 × g) for 1 min to elute the DNA

##9.	Performance verification and Fluorescence value determination
*	Transformed E. coli colonies are inoculated into liquid culture for plasmid extraction and subsequent DNA sequencing. After successful sequencing, it is used for induction experiments.

<center>Table 7 Configuration system</center>
|Component|Volume|
|:--|:--|
|Sugar solution of each substrate|100g/L| 
|Fluid medium|30ml| 
|Two-plasmid bacterial fluid|2ml| 
|Antibiotic|100μl| 
|PBS buffer|10ml| 

*	Steps:
1.	The concentration of each sugar solution is 100g/L, and after it is completely dissolved, it is filtered through a filter membrane in a biological safety cabinet.
2.	Add 50ul of ampicillin and streptomycin to the liquid medium and shake well.
3.	Add 2ml of the prepared bacterial solution containing the double plasmid to the liquid medium and shake well.
4.	Load the medium containing the bacterial solution and the two antibiotics into 10 shake tubes , and then label the type of substrate to be added, the temperature of the shaker to be placed in, and the concentration to be prepared, each temperature is labeled with 5 tubes according to the concentration gradient.
5.	Based on the concentration gradient, calculate the volume of sugar that needs to be added.
6.	According to the labeled information and the calculated volume, aspirate the same volume as the volume of substrate to be added the medium is then added to the substrate. Make sure that the total volume of 3 ml in each shake tube remains the same after the substrate is added.
7.	Place the shake tubes in a shaker at 25 °C and 37 °C for 16-24 h.
8.	Take 2 ml of the bacterial solution for centrifugation from each tube and then add 1ml PBS buffer and wash 2 times to reduce interference from background fluorescence.
9.	the fluorescence of the solution at an excitation wavelength of 488 nm and an emission wavelength of 518 nm was measured with a microplate reader value.

##10.   Microplate reader detection
1.	Plug in the power and turn on the power switch of the ELISA reader (press the switch key onthe back of the instrument), and turn on the switch connecting to the computer.
2.	Open the sofiware and enter the main operating interface of the software.
3.	Select "Start Measurement" and choose the corresponding measurement project and the layout of the plate being used.
4.	Click "OK", then click the "Start" button.
5.	At this point, the instrument is in measurement mode, and the operator should not perform any other operations.
6.	When the measurement is complete, the test results will be displayed on the monitor. Select the option to save the measurement data.
7.	Turn off the power of the ELISA reader.















